spec/apis/error-handlers.spec.js

```diff
@@ -171,4 +171,27 @@ describe("error handlers api", () => {
       ).toBeGreaterThan(-1);
     });
   });
+
+  it(`only throws one error when the application or parcel fails to mount`, async () => {
+    const app = {
+      async bootstrap() {},
+      async mount() {
+        throw Error("the mount failed");
+      },
+      async unmount() {
+        throw Error("the unmount failed");
+      }
+    };
+    location.hash = "#";
+    await singleSpa.triggerAppChange();
+    singleSpa.registerApplication("one-error-only", app, location =>
+      location.hash.startsWith("#one-error-only")
+    );
+    location.hash = "#one-error-only";
+    await singleSpa.triggerAppChange();
+
+    expect(errs.length).toBe(1);
```

Comment:
Before the code changes, `errs.length` was `2` instead of 1. One error for mount failure, and one for unmount failure.